src/services/auth.service.ts

```diff
@@ -83,3 +83,27 @@ export const loginUser = async (
     return { statusCode: 500, message: 'Internal server error' }
   }
 }
+
+// findOrCreateUser function for Google OAuth
+export const findOrCreateUser = async (profile: any): Promise<Profile> => {
```

Comment:
Add the type here

Reply:
when i add type for async (profile: any) i'm getting type error in passport.ts